R/remote.R

```diff
@@ -58,50 +58,61 @@ users_create_remote <- function(connect, prefix, expect = 1, check = TRUE, exact
     src = connect
   )
   message("Done creating remote users")
-  return(get_users(connect, prefix = prefix))
+  return(user_creation)
 }
 
 #' Create a Remote Group
 #'
-#' @param connect A R6 Connect object
-#' @param prefix character. The prefix of the group name to search for
-#' @param expect number. The number of responses to expect for this search
-#' @param check boolean. Whether to check for local existence first
+#' @param connect An R6 Connect object.
+#' @param prefix character. The prefix of the user name to search for.
+#' @param expect number. Optional. The number of responses to expect for this search.
+#' @param check boolean. Optional. Whether to check for local existence first.
+#' @param exact boolean. Optional. Whether to only create groups whose name
+#'   exactly matches the provided `prefix`.
 #'
-#' @return The results of creating the groups
+#' @return The results of creating the groups.
 #'
 #' @export
-groups_create_remote <- function(connect, prefix, expect = 1, check = TRUE) {
+groups_create_remote <- function(connect, prefix, expect = 1, check = TRUE, exact = FALSE) {
   expect <- as.integer(expect)
-  if (check && expect > 1) {
-    stop(glue::glue("expect > 1 is not tested. Please set expect = 1, and specify a more narrow 'prefix'. You provided: expect={expect}"))
-  }
   if (check) {
     # TODO: limit = 1 due to a paging bug in Posit Connect
```

Comment:
Doesn't have to be done in this PR, but would be good to `git blame` this comment and look to see if this issue still/actually exists in Connect.

Reply:
Created an issue to track: https://github.com/rstudio/connectapi/issues/319